src/openforms/submissions/logic/actions.py

```diff
@@ -40,6 +42,15 @@ def apply(self, step: SubmissionStep, configuration: JSONObject) -> None:
         """
         pass
 
+    def get_action_log_data(
+        self,
+        component_map: Dict[str, ComponentMeta],
+        all_variables: Dict[str, FormVariable],
+        initial_data: dict,
+    ) -> dict:
+        """Get action information to log"""
+        pass
```

Comment:
base implementation does not match type hint - I think I'd opt for `return None` as default implementation and change the type hint to be `-> Optional[JSONObject]`, as the return variable must be json-serializable.